### PR TITLE
Add missing upstream flags

### PR DESCRIPTION
**Pull request by catumin**
Some more flags have been added upstream, so including them here as well.

### CODE REVIEW

File: crates/hyfetch/src/presets.rs [hunk 1]
```diff
@@ -343,6 +383,10 @@ impl Preset {
                 "#8869A5", "#D8A7D8", "#FFFFFF", "#FDB18D", "#151638",
             ]),
 
+            Self::Biromantic2 => ColorProfile::from_hex_colors(vec![
```
Comment (teohhanhui) [inline crates/hyfetch/src/presets.rs]: Please remove this. This was just mislabeled greysexual flag, so it has been renamed in this rewrite.

File: crates/hyfetch/src/presets.rs [hunk 2]
```diff
@@ -443,6 +487,83 @@ impl Preset {
...
+                "#E50000", "#FF8D00", "#FFEE00", "#028121", "#004CFF", "#770088",
+            ]),
+
+            Self::Progress => ColorProfile::from_hex_colors(vec![
+                "#FFFFFF", "#FFAFC8", "#74D7EE", "#613915", "#000000", "#E50000", "#FF8D00",
+                "#FFEE00", "#028121", "#004CFF", "#770088",
+            ]),
+
+            Self::Intersex => ColorProfile::from_hex_colors(vec![
+                "#FFD800", "#FFD800", "#7902AA", "#FFD800", "#FFD800",
```
Comment (teohhanhui) [inline crates/hyfetch/src/presets.rs]: Please use weights where appropriate, i.e. whenever a colour is repeated consecutively.

File: crates/hyfetch/src/presets.rs (hunk 1 above)
Comment (catumin) [reply crates/hyfetch/src/presets.rs]: Ohhh okay, I see.

File: crates/hyfetch/src/presets.rs (hunk 2 above)
Comment (catumin) [reply crates/hyfetch/src/presets.rs]: Fixed in amended commit.

File: crates/hyfetch/src/presets.rs [hunk 3]
```diff
@@ -443,6 +481,82 @@ impl Preset {
...
+
+            Self::Solian => ColorProfile::from_hex_colors(vec![
+                "#FFF8ED", "#FFE7A8", "#F1B870", "#A56058", "#46281E",
+            ]),
+
+            Self::Lunian => ColorProfile::from_hex_colors(vec![
+                "#2F0E62", "#6F41B1", "#889FDF", "#7DDFD5", "#D2F2E2",
+            ]),
+
+            Self::Polyam => ColorProfile::from_hex_colors(vec![
```
Comment (teohhanhui) [inline crates/hyfetch/src/presets.rs:501]:
```suggestion
            // pulled from https://polyamproud.com/flag
            Self::Polyam => ColorProfile::from_hex_colors(vec![
```

File: crates/hyfetch/src/presets.rs [hunk 4]
```diff
@@ -443,6 +481,82 @@ impl Preset {
...
+                "#FFFFFF", "#FCBF00", "#009FE3", "#E50051", "#340C46",
+            ]),
+
+            Self::Sapphic => ColorProfile::from_hex_colors(vec![
+                "#FD8BA8", "#FBF2FF", "#C76BC5", "#FDD768", "#C76BC5", "#FBF2FF", "#FD8BA8",
+            ]),
+
+            Self::Androgyne => ColorProfile::from_hex_colors(vec!["#FE007F", "#9832FF", "#00B8E7"]),
+
```
Comment (teohhanhui) [inline crates/hyfetch/src/presets.rs:510]: FWIW, the Python source has a duplicate "genderflux" entry now at this position with different colours. We'll need to find out what happened and see what to do...

Comment (teohhanhui) [reply crates/hyfetch/src/presets.rs:510]: My investigation suggests this is the same flag as what we already have. The differences can be explained by inaccurate colour-picking.

@BKasin ~~Let's remove this?~~ Perhaps we could redo the colour-picking for the genderflux flag to make sure we have the most accurate colours? All the places I could find ultimately point back to the same source: https://www.deviantart.com/pride-flags/art/Genderflux-1-543925589

Comment (teohhanhui) [reply crates/hyfetch/src/presets.rs:510]: Confirmed that all the colours in our existing genderflux flag are as accurate as they can be.

**Comment by teohhanhui** [comment]
Thank you @BKasin :partying_face:

File: crates/hyfetch/src/presets.rs (hunk 4 above)
Comment (catumin) [reply crates/hyfetch/src/presets.rs:510]: I believe the repeat one was included with flags from a project called `hyfetch-extra-legal` that recently upstreamed their extra flags and it didn't get caught during that. The colour difference seems to be from using the same flag from different locations to get the source, so one may have had colours distorted by compression or something. I'm not sure which of the two is technically more accurate.

**Pull request merged**